README.md

```diff
@@ -47,20 +47,11 @@ You can now see the docs at `http://localhost:4567`.
 # Install Jinja for python templating.
 pip3 install Jinja2
 
-# Set your $GOPATH to a directory that has access to lncli.
-export GOPATH="/path/to/lnd"
-export PATH=$PATH:$GOPATH/bin
-
-# Install annotations and protoc-gen-doc.
-go get -u github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis
```

Comment:
This is still needed, but the go command won't work. We'll want to clone the repo instead to `$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway`. We should make sure not to clone it if it already exists too.

Reply:
I added the line back, but I think it's probably easier to just use the dockerized way in the future.